examples/routing_service/routing_service_constrained_wan/README.md

```diff
@@ -0,0 +1,3 @@
+These files are used to configure a constrained WAN demo that is available on RTI's community 'examples' website
```

Comment:
The build done in [Travis](https://travis-ci.org/github/rticommunity/rticonnextdds-examples/builds/690170925) is reporting some error related to the format of the MD file. Please, could you take a look and fix them?